engine/src/main/java/org/hibernate/search/query/dsl/impl/ConnectedMultiFieldsRangeQueryBuilder.java

```diff
@@ -116,12 +116,16 @@ private static Query createKeywordRangeQuery(String fieldName, RangeQueryContext
 		final Object fromObject = rangeContext.getFrom();
 		final Object toObject = rangeContext.getTo();
 		final DocumentBuilderIndexedEntity documentBuilder = Helper.getDocumentBuilder( queryContext );
-		final String fromString = fieldContext.objectToString( documentBuilder, fromObject, conversionContext );
+		final String fromString = fromObject == null ?
+				null :
+				fieldContext.objectToString( documentBuilder, fromObject, conversionContext );
 		final String lowerTerm = fromString == null ?
 				null :
 				Helper.getAnalyzedTerm( fieldName, fromString, "from", queryAnalyzer, fieldContext );
 
-		final String toString = fieldContext.objectToString( documentBuilder, toObject, conversionContext );
+		final String toString = toObject == null ?
```

Comment:
I am wondering whether we should consider the case where explicitly `above( null )` has been invoked. E.g. if the null token is "-1", that may make sense to exclude some negative values existing in the database for some reason. Maybe it's contrived, but I'd like to hear what others think.